package.json

```diff
@@ -45,11 +48,12 @@
     "express-interceptor": "1.2.0",
     "express-jwt": "5.0.0",
     "express-session": "1.13.0",
+    "http-status-codes": "^1.3.0",
```

Comment:
Just for understanding ? What if we want to support exact version ?

Reply:
We can use the actual version as well.. I used `npm install --save` which inherently uses the carat-based nearest version. Also, I think the cloud delivery principles would prefer this. Would check this and update the pull if required